nanofi/include/sitetosite/CPeer.h

```diff
@@ -79,15 +79,15 @@ static void setHostName(struct SiteToSiteCPeer * peer, const char * hostname) {
   peer->_host[host_len] = '\0';
   peer->_url[host_len + 7] = ':';
   if(peer->_port != 0) {
-    snprintf(peer->_url + host_len + 8, 5, "%d", peer->_port);
+    snprintf(peer->_url + host_len + 8, 6, "%d", peer->_port);
   }
   return;
 }
 
 static void setPort(struct SiteToSiteCPeer * peer, uint16_t port) {
   peer->_port = port;
   if(peer->_url != NULL) {
-    int i;
+    size_t i;
     for(i = strlen(peer->_url) -1; i >= 0; --i) {  // look for the last ':' in the string
```

Comment:
won't this result in an infinite loop?

Reply:
Good catch, thanks!  I have changed the type of `i` back to `int`.